gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -28,23 +28,25 @@
 
 /** Macros for accelerator types */
 #define ACCL_NONE_STR "none"
+#define ACCL_DEF_STR  "default"
 #define ACCL_AUTO_STR "auto"
 #define ACCL_CPU_STR  "cpu"
 #define ACCL_GPU_STR  "gpu"
 #define ACCL_NPU_STR  "npu"
 #define ACCL_NEON_STR "neon"
 #define ACCL_SRCN_STR "srcn"
-#define ACCL_DEF_STR  "default"
+#define ACCL_TPU_STR  "tpu"
 
 #define REGEX_ACCL_EACH_ELEM(ELEM) "[^!]" ELEM "[,]?"
 
 #define REGEX_ACCL_AUTO REGEX_ACCL_EACH_ELEM(ACCL_AUTO_STR)
+#define REGEX_ACCL_DEF  REGEX_ACCL_EACH_ELEM(ACCL_DEF_STR)
```

Comment:
Minor comment: it's better to move this line above `ACCL_AUTO_STR`?

Reply:
Done